templates/zerver/help/import-from-slack.md

```diff
@@ -1,149 +1,293 @@
     - `emoji:read`
     - `users:read`
     - `users:read.email`
     - `team:read`
 
-3. [Install the app](https://api.slack.com/authentication/basics#installing)
-  to your workspace. You will get an API token that you can now use to fetch
-      data from your Slack workspace.
+5. [Install your new app](https://api.slack.com/authentication/basics#installing)
+   to your Slack workspace.
+
+6. You will immediately see a **Bot User OAuth Token**, which is a long
+   string of numbers and characters starting with `xoxb-`. Copy this token. You
+   will use it to download user and emoji data from your Slack workspace.
 
 {end_tabs}
 
-### Export your Slack data
+### Import your data into Zulip
+
+To start using Zulip, you will need to choose between Zulip Cloud and
+self-hosting Zulip. For a simple
+managed solution, with no setup or maintenance overhead, you can [sign
+up](/new/) for Zulip Cloud with just a few clicks. Alternatively, you can
+[self-host](/self-hosting/) your Zulip organization. See
+[here](/help/getting-your-organization-started-with-zulip#choosing-between-zulip-cloud-and-self-hosting)
+to learn more.
 
-Now, [Export your Slack data](https://my.slack.com/services/export). You will
-receive a zip file `slack_data.zip`.
+#### Import into a Zulip Cloud organization
 
+If you plan to use Zulip Cloud, we'll take it from here! Please e-mail
+[support@zulip.com](mailto:support@zulip.com) with the following information:
 
-This step will also generate a different token starting with
-`xoxe-`; you don't need it.
+1. The subdomain you would like to use for your organization. Your Zulip chat will
+   be hosted at `<subdomain>.zulipchat.com`.
 
-### Import into Zulip Cloud
+    !!! warn ""
 
-Email support@zulip.com with `slack_data.zip`, the Slack API token
-generated above, and your desired subdomain. Your imported organization will
-be hosted at `<subdomain>.zulipchat.com`.
+        You can only import a Slack workspace as a new Zulip organization. If the organization already exists, the import process will overwrite all data that's already there. If needed, we're happy to preserve your data by moving an organization you've already created to a new subdomain prior to running the import process.
 
-If you've already created a test organization at
-`<subdomain>.zulipchat.com`, let us know, and we can rename the old
-organization first.
+1. The `zip` file containing your Slack message history export.
 
-### Import into a self-hosted Zulip server
+1. Your Slack **Bot User OAuth Token**, which will be a long
+   string of numbers and characters starting with `xoxb-`
 
-First
-[install a new Zulip server](https://zulip.readthedocs.io/en/stable/production/install.html),
-skipping "Step 3: Create a Zulip organization, and log in" (you'll
-create your Zulip organization via the data import tool instead).
 
-Copy the `slack_data.zip` file you obtained above onto the Zulip
+#### Import into a self-hosted Zulip server
```

Comment:
Reading this, I wonder if it's indeed best to keep this section here on the /help/ page or does this fit more as a ReadTheDocs doc and we could simply be linking to it here?

Reply:
Mm, I think that would be better, yeah. Thoughts on how to integrate it into ReadTheDocs.

I think it would be fine to merge this PR and make this change as a follow-up if it's a bit tricky.

Reply:
I went with using tabs for the two blocks, which really helps with one being more likely to see the "Import details" section below.

I think I like how the end result reads pretty well as-is, so don't feel a strong motivation to move something to ReadTheDocs.